src/widgets/AuthorBox.ts

```diff
@@ -9,8 +9,10 @@ export class GitAuthorForm
   extends Widget
   implements Dialog.IBodyWidget<Git.IIdentity>
 {
-  constructor() {
+  constructor(userName = '', userEmail = '') {
```

Comment:
Could we use an object here so it will be easier to extend and avoid the weird call if extended later as the one you saw to call `model.commit` that force to explicit argument that use their default values.

```suggestion
  constructor(options: {userName?: string, userEmail?: string} = {}) {
```

Reply:
I reused Git.IIdentity here and in some other places instead of (?) anonymous class. Let me know if you have any objections. I'm not TS experts, but it seems to be pretty clean now